packages/wpcom.js/.gitignore

```diff
@@ -0,0 +1,13 @@
+npm-debug.log
```

Comment:
Should we really include this file?

Reply:
As per your comment above, let's keep this file until we convert the data here into something else.